src/main/java/org/apache/datasketches/req/ReqSketch.java

```diff
@@ -235,6 +235,15 @@ public long getN() {
     return totalN;
   }
 
+  @Override
```

Comment:
Didn't realize `@Override` could appear above a comment like this. Makes sense in that comments are stripped when compiling.

Reply:
no comment